OWNERS.md

```diff
@@ -0,0 +1 @@
+troy@firstandthird.com
```

Comment:
I only added troy because he's the only one I knew worked on the entire thing.  People who work on the bulk of code would have their name here and you would add your name to a separate file on a folder level of what you've worked on, so that anyone coming in knows who is the correct person to ask when dealing with that code.